sdk/servicebus/service-bus/samples/javascript/usingAadAuth.js

```diff
@@ -30,19 +31,40 @@ require("dotenv").config();
 // Define Service Bus Endpoint here and related entity names here
 const serviceBusEndpoint =
   process.env.SERVICE_BUS_ENDPOINT || "<your-servicebus-namespace>.servicebus.windows.net";
+const queueName = process.env.QUEUE_NAME || "<queue name>";
 
 // Define CLIENT_ID, TENANT_ID and SECRET of your AAD application here
 const tenantId = process.env.AZURE_TENANT_ID || "<azure tenant id>";
 const clientSecret = process.env.AZURE_CLIENT_SECRET || "<azure client secret>";
 const clientId = process.env.AZURE_CLIENT_ID || "<azure client id>";
-async function main() {
+
+export async function main() {
+  if (
+    tenantId === "<azure tenant id>" ||
+    clientSecret === "<azure client secret>" ||
+    clientId === "<azure client id>"
+  ) {
+    console.log(
+      `Required environment variables are missing. Please ensure AZURE_TENANT_ID, AZURE_CLIENT_SECRET and AZURE_CLIENT_ID have been set.`
```

Comment:
We dont check for whether the queueName or namespace was provided, then why the check for these? Can we not rely on the error message from `DefaultAzureCredential`?

Reply:
The issue is they're not used so "npm run build:samples" fails because they're unreferenced.

I don't see any harm of checking them here since that's the entire point of the sample.